### DISCUSSION TITLE
Can profiles be compared?

### DISCUSSION THREAD

**Discussion by ManuelPeiro** [supermerill/SuperSlicer#2854] [General]
Hi,

During the time of use, I have accumulated quite a few print profiles in the three sections... Although I try to give them descriptive names, there are times when I have a hard time finding the differences between the print profiles.

Is there any way to compare them? in two columns of a spreadsheet, or word processor....

Thanks

**Comment by metasean** [comment]
<img width="922" alt="Compare presets button highlighted" src="https://user-images.githubusercontent.com/7552754/172976020-8e686bd8-8e60-42f3-82b3-9857f68051f3.png">

Yes, from the appropriate tab (i.e. "Print Settings", "Filament Settings", "Printer Settings"), just click this button highlighted above, and select the profiles that you want to compare.
